### DISCUSSION TITLE
Themepatcher

### DISCUSSION THREAD

**Discussion by rishav12s** [prasanthrangan/hyprdots#917] [Q&A] [answered]
can more themes be added to the hyprdots-mod repo and they will get added along with synthwave ? or does it have to be seperate branch/repos/folders for each theme ?

**Comment by kRHYME7** [comment, marked as answer]
+ It's not necessary to add on that repo, you can host it on your own

+ It will only patch themes one by one (Depending on theme name), but it will clone the whole repo. 

+  Follow the file tree which all configurations are in Config/ and the Gtk_Theme-Name.tar.gz source is inside Source\.  

+ Yes multiple themes are stored side by side, that is why Theme-Name should be consistent. 



Please read some of the discussions here. Someone already created some themes, you can get some inspiration to that.

**Reply by rishav12s** [reply]
yeah ig i'll try to do one and i've also used some of the themes using your hyprdots-ctl i really liked t-crypt's nordic blue theme and we have his themes on the main repo (two ig) so can you guys add to it to hyprdots-mod ?

**Reply by kRHYME7** [reply]
I think we can fork @T-Crypt's themes. But no need as we can just use the themepatcher on any repo.  

OWNER said that 10 is enough for the themes. So, more theme contributions will be on the hyprdots-mods.
 @rishav12s  But I think it's still good for you to host it to have realtime fixes. 

I'll be happy to use your themes! Let as know when you're done.

**Reply by kRHYME7** [reply]
BTW you can just fork this repo and build your own theme with specific Theme-name, we can still use the patcher for that. Though it will take time to download as we are downloading the whole repo.

**Reply by rishav12s** [reply]
i see thanks gonna try to make one later
